test/e2e/tests/communities/test_communities_permissions.py

```diff
@@ -184,7 +183,6 @@ def test_add_edit_and_remove_permissions(main_screen: MainWindow, params):
 
 
 @pytest.mark.parametrize('params', [constants.community_params])
-@pytest.mark.skip(reason="https://github.com/status-im/status-desktop/issues/15246")
 def test_add_5_member_role_permissions(main_screen: MainWindow, params):
```

Comment:
```suggestion
@pytest.mark.critical
```
added before the function definition will enable the test for PR level